roboflow/core/project.py

```diff
@@ -101,31 +102,23 @@ def __image_upload(self, image_path, hosted_image=False, split="train"):
             image_name = os.path.basename(image_path)
 
             # Construct URL for local image upload
-
             self.image_upload_url = "".join([
                 "https://api.roboflow.com/dataset/", project_name, "/upload",
-                "?api_key=", self.__api_key,
-                "&name=" + image_name,
-                "&split=" + split
```

Comment:
@patrick-morgan @hansent did we ablate split and name because things changed in the upload api with the new label flow? otherwise i'll add them back in before deploy

Reply:
Yeah, those parameters are no longer used in the URL for multipart form uploading. That data (split and name) is now sent in the request on line 120. Good that you pointed that out though as I will update the documentation to make that more clear